src/components/SuperDrawer/useSuperDrawer.tsx

```diff
@@ -26,8 +18,6 @@ export interface OpenInDrawerOpts {
 }
 
 export interface OpenDetailOpts {
-  /** Title of the SuperDrawer for this detail page (replaces the original title). */
-  title?: string;
```

Comment:
Question: So components opened within `openDetail` will be responsible for bringing their own `SuperDrawerHeader` now? I think that makes sense, just checking my understanding.

Reply:
Yes, that is correct.